apps/class-solid/src/routes/about.tsx

```diff
@@ -1,17 +1,14 @@
 import { A } from "@solidjs/router";
-import Counter from "~/components/Counter";
```

Comment:
Ah I made similar change in https://github.com/classmodel/class-web/pull/8/commits/a4549ac03cc7f52bc6424351673f6334ab24e405, we will need to resolve that conflict later.